app/image_ai/read_image.py

```diff
@@ -9,17 +9,14 @@
 # print("Current Directory:", os.getcwd())
 
 # Load all card images from the folder into an array
-cards_dir = os.path.join(os.getcwd(), 'app', 'image_ai', 'card_images_large_screen_h_small')
+cards_dir = os.path.join(os.getcwd(), 'app', 'image_ai', 'card_images_800x600')
 cards_images = []
 found_cards = []  # List to store matched cards
 found_cards_dealer = []  # List to store matched cards
 attempt_counter = 1 ##counter of mismatch
 last_message_is_not_found = False
 
-roi_configurations = ["LARGE_SCREEN", "SMALL_SCREEN", "ALTERNATIVE_MONITOR"]
-
-
-
+roi_configurations = ["LARGE_SCREEN", "SMALL_SCREEN", "ALTERNATIVE_MONITOR","800v600"]
```

Comment:
don't need to be done now but lets move all constants for the image search into a constants file so we can easily change the config without having to read through the logic